package.json

```
@@ -41,14 +41,17 @@
     "remark-stringify": "^9.0.0",
     "tape": "^5.0.0",
     "tinyify": "^3.0.0",
+    "type-coverage": "^2.0.0",
+    "typescript": "^4.0.0",
     "unified": "^9.0.0",
     "xo": "^0.36.0"
   },
   "scripts": {
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "build-types": "tsc -b",
```

Could type coverage be run as part of the build/test script?
```suggestion
    "build-types": "tsc -b && type-coverage",
```
It  may need a configuration like: https://github.com/syntax-tree/mdast-util-to-markdown/blob/d51bf747f55bdc383e689051abff80c63b0ca5f5/package.json#L89-L93